### ISSUE TITLE
Some feature names collide with Action routes

### ISSUE THREAD

**Issue by carlthuringer** [jnunemaker/flipper#80]
I ran into an issue implementing my first feature, `refactor-images`. The application would crash with a bizarre error: `Flipper::UI::RequestMethodNotSupported - Flipper::UI::Actions::File does not support request method "post"`
This happened when I tried to add a specific actor using the UI.

This made me think that I had broken it somehow, or otherwise set it up incorrectly, but my only mistake was treading upon [A Regular Expression](https://github.com/jnunemaker/flipper/blob/master/lib/flipper/ui/actions/file.rb#L9)

I discovered my error by the expedient combo of debugging, stepping, and `print @action_collection.method(:action_for_request).source`. But of course I think more common consumers of the gem may be completely confused and give up or post an issue.

Shouldn't you specify the route match more strictly, rather than matching any part of the full path? 
```ruby
route %r{\a#{base_path}(images|css|js|octicons|fonts)/.*\Z}
```
Else, no features containing these five words will be permitted.

**Comment by jnunemaker** [comment]
That stinks. Sorry. Added a spec and fixed it by adjusting the order of the action collection and thus route processing. Released 0.7.1 which should solve your issue.

**Comment by jnunemaker** [comment]
Oh, big thanks for tracking it down too. Made fixing it easy.
